tests/functional/openstackoperator_controller_test.go

```diff
@@ -181,6 +183,19 @@ var _ = Describe("OpenStackOperator controller", func() {
 			// make keystoneAPI ready and create secrets usually created by keystone-controller
 			keystone.SimulateKeystoneAPIReady(names.KeystoneAPIName)
 
+			// openstackversion exists
+			Eventually(func(g Gomega) {
+				osclient := GetOpenStackVersion(names.OpenStackControlplaneName)
```

Comment:
nit: osversion

Reply:
I will fix this since I have to rebase anyway.